proto/api/v1/search_service.proto

```diff
@@ -8,68 +8,69 @@ import "api/v1/pagination.proto";
-    COMPLIANCE_RESULTS        = 43;
-    COMPLIANCE_DOMAIN         = 44;
-    CLUSTER_HEALTH            = 45;
-    POLICY_CATEGORIES         = 46;
-    IMAGE_INTEGRATIONS        = 47;
-    COLLECTIONS               = 48;
-    POLICY_CATEGORY_EDGE      = 49;
-    PROCESS_LISTENING_ON_PORT = 50;
-    API_TOKEN                 = 51;
-    REPORT_METADATA           = 52;
-    REPORT_SNAPSHOT           = 53;
-    COMPLIANCE_INTEGRATIONS   = 54;
-    COMPLIANCE_SCAN_SETTINGS  = 55;
-    COMPLIANCE_SCAN           = 56;
-    COMPLIANCE_CHECK_RESULTS  = 57;
-    BLOB                      = 58;
-    PRODUCT_USAGE             = 59;
-    ADMINISTRATION_EVENTS     = 60;
+    SEARCH_UNSET                   = 0;
+    ALERTS                         = 1;
+    IMAGES                         = 2;
+    IMAGE_COMPONENTS               = 20;
+    IMAGE_VULN_EDGE                = 27;
+    IMAGE_COMPONENT_EDGE           = 23;
+    POLICIES                       = 3;
+    DEPLOYMENTS                    = 4;
+    ACTIVE_COMPONENT               = 30;
+    PODS                           = 25;
+    SECRETS                        = 5;
+    PROCESS_INDICATORS             = 6;
+    COMPLIANCE                     = 7;
+    CLUSTERS                       = 8;
+    NAMESPACES                     = 9;
+    NODES                          = 10;
+    NODE_COMPONENTS                = 38;
+    NODE_VULN_EDGE                 = 28;
+    NODE_COMPONENT_EDGE            = 29;
+    NODE_COMPONENT_CVE_EDGE        = 33;
+    COMPLIANCE_STANDARD            = 11;
+    COMPLIANCE_CONTROL_GROUP       = 12;
+    COMPLIANCE_CONTROL             = 13;
+    SERVICE_ACCOUNTS               = 14;
+    ROLES                          = 15;
+    ROLEBINDINGS                   = 16;
+    REPORT_CONFIGURATIONS          = 37;
+    PROCESS_BASELINES              = 17;
+    SUBJECTS                       = 18;
+    RISKS                          = 19;
+    VULNERABILITIES                = 21;
+    CLUSTER_VULNERABILITIES        = 34;
+    IMAGE_VULNERABILITIES          = 35;
+    NODE_VULNERABILITIES           = 36;
+    COMPONENT_VULN_EDGE            = 22;
+    CLUSTER_VULN_EDGE              = 24;
+    NETWORK_ENTITY                 = 26;
+    VULN_REQUEST                   = 32;
+    NETWORK_BASELINE               = 39;
+    NETWORK_POLICIES               = 40;
+    PROCESS_BASELINE_RESULTS       = 41;
+    COMPLIANCE_METADATA            = 42;
+    COMPLIANCE_RESULTS             = 43;
+    COMPLIANCE_DOMAIN              = 44;
+    CLUSTER_HEALTH                 = 45;
+    POLICY_CATEGORIES              = 46;
+    IMAGE_INTEGRATIONS             = 47;
+    COLLECTIONS                    = 48;
+    POLICY_CATEGORY_EDGE           = 49;
+    PROCESS_LISTENING_ON_PORT      = 50;
+    API_TOKEN                      = 51;
+    REPORT_METADATA                = 52;
+    REPORT_SNAPSHOT                = 53;
+    COMPLIANCE_INTEGRATIONS        = 54;
+    COMPLIANCE_SCAN_SETTINGS       = 55;
```

Comment:
nit : Can this be renamed to COMPLIANCE_SCAN_CONFIG to match the storage type?

Reply:
nice find.